### PR TITLE
Made grammar easier to understand.

### PR DESCRIPTION
**Pull request by kapeyka**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by tbushman** [comment]
@kapeyka None of the capitalizations / Uppercases are necessary, and two of the changes are incorrect.

File: client/src/pages/learn/coding-interview-prep/algorithms/index.md [hunk 1]
```diff
@@ -5,16 +5,16 @@ superBlock: Coding Interview Prep
...
 
 This course presents you with the most asked questions during the interviews to prepare your basic knowledge of Algorithms.
 It will provide a clear and concise understanding and implementations of different Algorithms.
 
-# Prerequisite: All the Algorithmic questions would require the solutions in Javascript. So, it's a good idea to complete the javascript course first.
+# Prerequisite: All the Algorithmic questions would require the solutions in Javascript. So, it's a good idea to complete the Javascript course first.
 
-It's a very good practise to use a whiteboard or a notepad to implement and practise the interview questions. Since most of the coding interviews use either a Whiteboard or a Notepad for writing the codes. So forget your nice IDEs and debuggers and start getting your hands dirty.
+It's a very good practice to use a Whiteboard or a Notepad to implement and practice the interview questions, since most coding interviews use either a Whiteboard or a Notepad for writing the code. So forget your nice IDEs and debuggers and start getting your hands dirty.
```
Comment (tbushman) [inline client/src/pages/learn/coding-interview-prep/algorithms/index.md]: Spelling correction -- practise to practice. Capitalization of `Javascript` also good.

Comment (akkatracker) [reply client/src/pages/learn/coding-interview-prep/algorithms/index.md]: I would argue it sounds more correct to say `It's very good practice` i.e. removing the a

**Comment by cmccormack** [comment]
With the additional changes, this PR looks good to me.  Thank you for your contribution!

**Pull request merged**